.github/workflows/add-labels.yml

```diff
@@ -1,6 +1,10 @@
 name: Add Labels
 
-on: [issues, pull_request]
```

Comment:
Quick Q. The old config applies to default action types only, hence we updated with some customized types?